### PR TITLE
2525C Legacy Edits

### PR DESCRIPTION
**Pull request by Dbarnes1**

### CODE REVIEW

**Comment by Dbarnes1** [comment]
@abouffard these are the final edits I made to the 2525C style. Nothing in here fixes the weird text label offset, but if someone else wanted to build the legacy stylx with all the fixes they would need these changed files

File: instance/jmsml_D_Land_Civilian.xml [hunk 1]
```diff
@@ -289,6 +289,9 @@
     </LegacySymbol>
     <LegacySymbol ID="CIVILIAN_ORGANIZATION_TARGETED_MOD_SYM" Label="O*GPC-----*****" EntityID="CIVILIAN" EntityTypeID="ORGANIZATION" ModifierOneID="TARGETED_MOD">
       <LegacyFunctionCode Name="2525C" SchemaOverride="O" DimensionOverride="G">C-----</LegacyFunctionCode>
+	  <LegacySymbol ID="CIVILIAN_ORGANIZATION_TARGETED_MOD_LE" Label="STBOPS.GRPORG.TERRST" Graphic="O-G-C-----.svg">
+		<LegacyFunctionCode Name="2525C" SchemaOverride="O" DimensionOverride="G">C-----</LegacyFunctionCode>
+	  </LegacyEntity>
```
Comment (abouffard) [inline instance/jmsml_D_Land_Civilian.xml]: @Dbarnes1 - Elements are not closed properly here.

File: instance/jmsml_D_Land_Unit.xml [hunk 2]
```diff
@@ -2689,8 +2692,10 @@
       <LegacyFunctionCode Name="2525C">UCFTCD</LegacyFunctionCode>
 	    <LegacyEntity ID="FIRES_FIELD_ARTILLERY_OBSERVER_LE" Label="WAR.GRDTRK.UNT.CBT.FLDART.TGTAQ.CLT" Icon="FULL_FRAME" CloverGraphic="S-G-UCFTC-_0.svg" RectangleGraphic="S-G-UCFTC-_1.svg" SquareGraphic="S-G-UCFTC-_2.svg" DiamondGraphic="S-G-UCFTC-_3.svg">
 		    <LegacyFunctionCode Name="2525C">UCFTC-</LegacyFunctionCode>
-        <LegacyFunctionCode Name="2525C">UCFTCD</LegacyFunctionCode>
 	    </LegacyEntity>
+		<LegacyEntity ID="FIRES_FIELD_ARTILLERY_OBSERVER_LE" Label="WAR.GRDTRK.UNT.CBT.FLDART.TGTAQ.CLT.DMD" Icon="FULL_FRAME" CloverGraphic="S-G-UCFTCD_0.svg" RectangleGraphic="S-G_UCFTCD_1.svg" SquareGraphic="S-G-UCFTCD_2.svg" DiamondGraphic="S-G-UCFTCD_3.svg">
```
Comment (abouffard) [inline instance/jmsml_D_Land_Unit.xml]: @Dbarnes1 - needs to be unique and needs indenting

**Comment by Dbarnes1** [comment]
@abouffard should be ready to merge

**Pull request merged**